packages/flutter/lib/src/widgets/form.dart

```diff
@@ -635,4 +697,13 @@ enum AutovalidateMode {
   /// Used to auto-validate [Form] and [FormField] only after each user
   /// interaction.
   onUserInteraction,
+
+  /// Used to auto-validate current [FormField] when unfocused.
+  ///
+  /// This mode is useful when you want to validate a [FormField] after the user
+  /// has interacted with another one.
+  ///
+  /// If you want validate all fields of a [Form] after the user has interacted
```

Comment:
See this part of the styleguide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple